wagtail/admin/templates/wagtailadmin/shared/status_tag.html

```diff
@@ -0,0 +1 @@
+<span class="status-tag {{classname}}">{{children}}</span>
```

Comment:
Just check your editor settings - looks like your files are being saved without a line break  at the end.

Reply:
no
actually that caused an error in tests

Reply:
![image](https://user-images.githubusercontent.com/74553951/207445061-ef3f9a10-fb57-4442-9e51-cbe12fd9b5f5.png)
 snippet's test was looking for exact one line html while status tag was injecting the one with line breaks

Reply:
so i had to put the status tag partial's html in one to pass the test

Reply:
Ok - you can use spaceless to solve most of this problem. 

https://docs.djangoproject.com/en/4.1/ref/templates/builtins/#spaceless

We will also want to add some docs to the template file so maybe look at the icon template html is done as a reference (it also accepts classname).

Reply:
i just checked through the snippet test its still passing even with the line break, but there's been a multiple cases where status tag has been used in single line between other tags , i am little confused about it should we still use ``` spaceless```?

Reply:
Avoid spaceless unless you need it. And if it's only one or two tests that fail - sometimes it's best to refine the test assertion to not be so strict.

Reply:
alright then its good to go there's no test failing on that scope